src/include/storage/ducklake_metadata_info.hpp

```diff
@@ -184,6 +184,8 @@ struct DuckLakeSnapshotInfo {
 	timestamp_tz_t time;
 	idx_t schema_version;
 	SnapshotChangeInfo change_info;
+	string author;
```

Comment:
Should this not be a `Value` so we can distinguish between `NULL` and the empty string?